atkinson/errors/reporters/trello.py

```diff
@@ -0,0 +1,216 @@
+#! /usr/bin/env python
+""" Trello Card error reporter """
+
+import re
+
+from trollo import TrelloApi
+
+from atkinson.config.manager import ConfigManager
+from atkinson.errors.reporters import BaseReport
+
+
+def get_trello_api():
+    """
+    Construct an interface to trello using the trollo api
+    """
+    conf = ConfigManager('trello.yml').config
+    return TrelloApi(conf['api_key'], conf['token'])
+
+
+def get_columns(api, config):
+    """
+    Get the columns for a trello board given a api handle and the id
+    of the board.
+
+    :param api: A trollo Boards api handle.
+    :param dict config: The trello unique id for the board to fetch the data.
+    :return: A tuple of column trello ids
+    """
+    for key in ['board_id', 'new_column', 'close_column']:
+        if key not in config:
+            raise KeyError(f"A required key '{key}' is missing in the config")
+
+    column_data = {x['name']: x['id']
+                   for x in api.get_list(config['board_id'])}
+    return (column_data[config['new_column']],
+            column_data[config['close_column']])
```

Comment:
This function converts a string name for a column into it's trello id. For example a column called 'New' may have an id like 'abc12345' from trello. The user only needs to know the user readable string name and this method will obtain the rest. I updated the doc string to try and make this more clear and I hope it helps.

I could make this function module scoped since it's use outside of this module would be limited. I did not do that at first so I could better test it. However it may make sense to do that and go about  testing it another way.